packages/ui/src/machines/authenticator/defaultServices.ts

```diff
@@ -78,12 +146,11 @@ export const defaultServices = {
       (password || confirm_password) &&
       password !== confirm_password &&
       ((touched_confirm_password && touched_password) ||
-        (password.length >= 6 && confirm_password.length >= 6))
+        (password?.length >= 6 && confirm_password?.length >= 6))
```

Comment:
This was a bug I found and fixed. This would silently fail if the password didn't exist.